apps/condo/domains/ticket/utils/clientSchema/IncidentRenders.tsx

```diff
@@ -134,8 +136,9 @@ export const getManyIncidentClassifiersGroupByPlaceRender = (ellipsis?: boolean
         const problemsPart = !isEmpty(problemNames) ? ` » ${problemNames.join(', ')}` : ''
 
         const text = `${categoriesPart}${problemsPart}`
+        const trimmedText = text.length > MAX_CELL_CONTENT_LENGTH ? `${text.substring(0, MAX_CELL_CONTENT_LENGTH)}…` : text
 
-        return getTableCellRenderer(null, ellipsis)(text)
+        return getTableCellRenderer(null, false, null, null, null, text)(trimmedText)
```

Comment:
I think here we also need to use the object parameter

Reply:
I'm afraid to refactor it))

Reply:
It is worth starting a new task for this